test/component/uk/gov/hmrc/exports/base/ComponentTestSpec.scala

```diff
@@ -102,34 +108,14 @@ trait ComponentTestSpec
     )
 
   def verifySubmissionRepositoryIsCorrectlyCalled(eoriValue: String) {
-    val submissionCaptor: ArgumentCaptor[Submission] = ArgumentCaptor.forClass(classOf[Submission])
-    verify(mockSubmissionRepository).save(submissionCaptor.capture())
-    submissionCaptor.getValue.eori shouldBe eoriValue
+//    val submissionCaptor: ArgumentCaptor[Submission] = ArgumentCaptor.forClass(classOf[Submission])
+//    verify(mockSubmissionRepository).save(submissionCaptor.capture())
+//    submissionCaptor.getValue.eori shouldBe eoriValue
```

Comment:
Still method that contains commented code. Please remove whole method, if you need this method add return type and uncomment code

Reply:
Removed - this class is unused and will be removed